zstd-safe/zstd-sys/build.rs

```diff
@@ -5,8 +5,10 @@ use std::{env, fs};
 #[cfg(feature = "bindgen")]
 fn generate_bindings(defs: Vec<&str>, headerpaths: Vec<PathBuf>) {
     let bindings = bindgen::Builder::default()
-        .header("zstd.h")
-        .header("zdict.h")
+        .header("zstd.h");
+    #[cfg(feature = "zdict")]
+    let bindings = bindings.header("zdict.h");
+    bindings
```

Comment:
```suggestion
    let bindings = bindings
```
Otherwise it will consume `bindings` (and probably not compile) and not actually update it in-place.